website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Edit Permissions | PUT
+        Rule([
+            '/project/<pid>/wiki/permissions/<permissions>/',
+            '/project/<pid>/node/<nid>/wiki/permissions/<permissions>/',
+        ], 'put', views.edit_wiki_permissions, json_renderer),
+
+        #Permissions Info for Settings Page | GET
+        Rule(
+            [
+                '/project/<pid>/wiki/permissions/',
```

Comment:
This doesn't seem like an appropriate URL and view name because the view actually returns more than just permissions--it returns a treebeard-formatted representation of a node's wiki settings.

Thoughts, @brianjgeiger on an appropriate URL pattern?